src/Microsoft.IdentityModel.Logging/InternalsVisibleTo.cs

```diff
@@ -1,8 +1,13 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Microsoft.IdentityModel.JsonWebTokens, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
```

Comment:
Can we compile include default telemetry implementation to avoid friend assemblies?

Reply:
It looks like doing this would mean including the interface, the implementation, the constants, IdentityModelTelemetry, and adding them all to that assembly's api file.

Reply:
Adding new friend assemblies is definitely not a good ideal direction. Can we compile in these 3? They seem lightweight.